Conformance results for v1.27/metakube-openstack

ovstuckrad opened the pull request:
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] If your product/project is open source, did you include the `repo_url`?
* [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

For a full list of requirements, please refer to these sections of the docs: [_Contents of the PR_](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr), and [_Requirements_](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#requirements).

taylorwaggoner commented:
@ovstuckrad thanks for submitting. Could you please add a bit more detail to your README about what is asked during the cluster wizard? The goal is to provide enough details here that a user could recreate your test results. thanks!

ovstuckrad commented:
Hi @taylorwaggoner & thank you for checking. I have added some more information on the cluster creation and also added a link to our in-depth documentation. I hope this clarifies the steps to recreate the test better.
